src/NodeAutoCompleteViewExtension/ViewModels/NodeAutoCompleteBarViewModel.cs

```diff
@@ -236,11 +231,10 @@ internal void ConsolidateTransientNodes()
                 connector.IsConnecting = false;
             }
 
-            NodeAutoCompleteUtilities.PostAutoLayoutNodes(node.WorkspaceViewModel.Model, node.NodeModel, transientNodes.Select(x => x.NodeModel), true, true, false, null);
```

Comment:
Why remove this ? When doing the consolidation we post a final autolayout that will account for the overalapping nodes in the area that do not belong to the cluster. For transient cluster we must not touch the existing nodes indeed but at the end I think we should have it.

Reply:
Oh is it because of the undo involved by the autolayout ?

Reply:
I actually think I can add it back. I must’ve removed it while trying to get the undos in line.